LeagueOfTowers/Assets/Scripts/CameraMovement.cs

```diff
@@ -0,0 +1,88 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraMovement : MonoBehaviour
+{
+    [SerializeField] private float moveSpeed;
+    // Start is called before the first frame update
+
+    private float minX;
+    private float maxX;
+    private float minY;
+    private float maxY;
+
+    private void Awake()
+    {
+       
+    }
+
+
+    void Start()
+    {
+        /*this.minX = mapMan.getMapBounds()[0];
```

Comment:
If this commented, do we need it to stay?

Reply:
yeah i will delete those.